### PR TITLE
partner_education_views: fixed display name

### PR DESCRIPTION
**Pull request by SebastianGieser-Vertel**

### CODE REVIEW

**Comment by FredrikArvas** [comment]
Please add change and description in the manifest

File: partner_education_views/models/education_level.py [hunk 1]
```diff
@@ -7,3 +7,16 @@ class ResPartnerEducationLevel(models.Model):
 
     name = fields.Integer(string="Education level")
     description = fields.Char(string="Description")
+
+    @api.multi
+    def name_get(self):
+        res = super(ResPartnerEducationLevel, self).name_get()
+        data = []
+        for edu_lvl in self:
```
Comment (nils-vertel) [inline partner_education_views/models/education_level.py]: String concatenation is pretty expensive, its cheaper to build a list and then do "".join(my_list)
In this case it does probably not matter but think of it in the future.

File: partner_education_views/models/education.py [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
+    def name_get(self):
+        res = super(ResPartnerEducation, self).name_get()
+        data = []
+        for education in self:
+            display_value = ''
+            display_value += education.sun_id.name or ""
+            display_value += ', '
+            display_value += education.education_level_id.name or _("None")
+            data.append((education.id, display_value))
+        return data
```
Comment (nils-vertel) [inline partner_education_views/models/education.py]: Missing newline at end of file.

File: partner_education_views/models/education.py [hunk 3]
```diff
@@ -1,14 +1,27 @@
...
     foreign_education_approved = fields.Boolean(
-        string="Foreign education approved")
\ No newline at end of file
+        string="Foreign education approved")
+
+    @api.multi
+    def name_get(self):
+        res = super(ResPartnerEducation, self).name_get()
+        data = []
+        for education in self:
```
Comment (nils-vertel) [inline partner_education_views/models/education.py]: String concatenation is expensive a better method would be to construct a list and then join it.
Various ways to do this:
 "".join(my_list)
"".join(["string1" or "", string2 or ""])
f'{education.sun_id.name or ""},{education.education_level_id.name or _("None")}'

**Review by nils-vertel — APPROVED** [review]

File: partner_education/models/education_level.py [hunk 4]
```diff
@@ -7,3 +7,14 @@ class ResPartnerEducationLevel(models.Model):
 
     name = fields.Integer(string="Education level")
     description = fields.Char(string="Description")
+
+    @api.multi
+    def name_get(self):
+        res = super(ResPartnerEducationLevel, self).name_get()
```
Comment (daerio) [inline partner_education/models/education_level.py:13]: We're not using res. Is there a reason to call this super? If not, remove it.

File: partner_education/models/education.py [hunk 5]
```diff
@@ -13,3 +13,14 @@ class ResPartnerEducation(models.Model):
     foreign_education = fields.Boolean(string="Foreign education")
     foreign_education_approved = fields.Boolean(
         string="Foreign education approved")
+
+    @api.multi
+    def name_get(self):
+        res = super(ResPartnerEducation, self).name_get()
```
Comment (daerio) [inline partner_education/models/education.py:19]: We're not using res. Is there a reason to call this super? If not, remove it.

**Review by daerio — CHANGES_REQUESTED** [review]
see comments

**Comment by SebastianGieser-Vertel** [comment]
This code will be moved to a different branch.

**Pull request closed without merging**